sympy/integrals/integrals.py

```diff
@@ -28,6 +28,8 @@
 from sympy.series.order import Order
 from sympy.series.formal import FormalPowerSeries
 from sympy.simplify.fu import sincos_to_sum
+from sympy.calculus import singularities
```

Comment:
This should be `from sympy.calculus.singularities import singularities`. (`singularities` is both a module and a function.)

Reply:
@jksuom  Yeah, I figured that out too. Thanks anyways.